distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -45,20 +45,6 @@ Before searching in ${branding}, at least one workspace must be created.
 .Start new workspace
 image::start-new-workspace.png[start new workspace icon]
```

Comment:
❓ Should this line also be removed if we are removing `start-new-workspace.png`?

Reply:
I didn't remove - only edited https://github.com/codice/ddf/blob/6f6cb583b576ed4a4ee988da24376289afc23f23/distribution/docs/src/main/resources/images/start-new-workspace.png